OracleHTTPServer/dockerfiles/12.2.1/Dockerfile

```diff
@@ -66,6 +66,8 @@ RUN yum install -y \
         unzip tar nfs-utils binutils compat-libcap1 compat-libstdc++ \
         gcc gcc-c++ glibc glibc-devel libaio libaio-devel libgcc \
 	libstdc++ libstdc++-devel ksh make ocfs2-tools sysstat && \
+    yum clean all && \
+    rm -rf /var/cache/yum/* && \
```

Comment:
Certainly, [Yum documentation](https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/6/html/Deployment_Guide/sec-Working_with_Yum_Cache.html#idp30184448) indicates that `yum clean all` removes entries accumulated in the `/var/cache/yum/` directory. However, that is not always true. There are [hundreds of Dockerfiles out there](https://www.google.com/search?q=Dockerfile+%22yum+clean+all%22+%22rm+-rf+/var/cache/yum/*%22+site:github.com) that still use this little tick (`rm -rf /var/cache/yum/*`) to completely clear Yum cache.